src/cmd/manage_nsfs.js

```diff
@@ -287,24 +288,25 @@ async function fetch_account_data(argv, config_root, from_file) {
     if (!data) {
         const name = argv.name && new SensitiveString(String(argv.name));
         const email = argv.email && new SensitiveString(String(argv.email));
-        const uid = Number(argv.uid) || process.getuid();
-        const gid = Number(argv.gid) || process.getgid();
+        const distinguished_name = argv.user && new SensitiveString(String(argv.user));
```

Comment:
let's add cli tests in test_nc_nsfs_cli.js

Reply:
done